doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,102 @@
+# Symfony forms content edit
```

Comment:
I wouldn't merge this document as is.

Could you try to update it / write a shorter one that covers what is *actually* in *this* PR ?